apps/docs-e2e/src/docs.spec.ts

```diff
@@ -10,35 +12,178 @@ const test = base.extend<{ docsLayoutPage: DocsLayout }>({
   },
 })
 
-const DOCS_SITE_URL = 'http://localhost:3000/docs'
-const CUHACKING_2025_PLATFORM_GITHUB_REPOSITORY_URL = 'https://github.com/cuhacking/2025'
-const CUHACKING_2025_PLATFORM_GITHUB_PROJECT_BOARD_URL = 'https://github.com/orgs/cuhacking/projects/4'
+const GITHUB_BASE_URL = 'https://github.com/cuhacking'
+const GITHUB_ORG_BASE_URL = 'https://github.com/orgs/cuhacking'
+const DOCS_BASE_URL = 'http://localhost:3000'
 
-test('should contain page title', async ({ docsLayoutPage }) => {
-  await expect(docsLayoutPage.page).toHaveTitle(/Welcome to the Docs/)
-})
+const CUHACKING_2025_PLATFORM_GITHUB_PROJECT_BOARD_URL = `${GITHUB_ORG_BASE_URL}/projects/4`
+const CUHACKING_2025_PLATFORM_GITHUB_REPOSITORY_URL = `${GITHUB_BASE_URL}/2025`
+const CUHACKING_2025_PLATFORM_GITHUB_INDEX_PAGE_URL = `${GITHUB_BASE_URL}/2025/blob/main/apps/docs/src/content/docs/index.mdx`
+const CUHACKING_2025_LANDING_PAGE_GITHUB_REPOSITORY_URL = `${GITHUB_BASE_URL}/landing-page`
+
+const CUHACKING_2025_DOCS_URL = `${DOCS_BASE_URL}/docs`
+
+const CUHACKING_2025_LANDING_PAGE_URL = 'https://www.cuhacking.ca/'
 
-test.describe('should contain desktop header elements', {
+/* ---------------- MOBILE + DESKTOP + TABLET ---------------- */
+test.describe(`Common MOBILE, TABLET and DESKTOP Layout Elements`, {
   tag: '@smoke',
 }, () => {
-  test('should contain cuHacking logo icon in desktop header', async ({ docsLayoutPage }) => {
+  test(`should contain title page`, async ({ docsLayoutPage }) => {
```

Comment:
is there something similar to this: `  test.use({ viewport: devices['Pixel 5'].viewport });`. I just saw this on stackoverflow, not sure if it's correct. 

Or we can have TWO pipelines one for mobile and desktop, then we can use the `--grep`/`--g` flag to run mobile tests only and desktop tests only. What are your thoughts?

Reply:
I don't know if I follow. I had these tests setup with the viewports already and changed it to work with projects. Are you asking that I revert these changes?

Reply:
^^

Reply:
Wouldn't our pipeline always fail? Because the pipeline runs the tests on all the projects? Correct me if I'm wrong, if i'm completely misunderstanding playwright, I'll approve the PR.

Reply:
you are right, i now have made according changes to the code, but some tests still fail. I think the pipeline is setup so that it runs every test at once, but this creates a lot of overhead which results in a couple of tests timing out (even though they run fine individually and are correct). I'll look into this more...